readthedocs/oauth/services/gitlab.py

```diff
@@ -539,16 +534,16 @@ def send_build_status(self, build, commit, state):
             if resp.status_code == 201:
                 log.info(
                     "GitLab commit status created for project: %s, commit status: %s",
-                    project,
+                    project.slug,
                     gitlab_build_state,
                 )
                 return True
 
             if resp.status_code in [401, 403, 404]:
                 log.info(
-                    'GitLab project does not exist or user does not have '
-                    'permissions: project=%s',
-                    project,
+                    'GitLab project does not exist or user does not have permissions: '
```

Comment:
We should probably make sure our logging is consistent here across all providers. I've definitely wanted better logging here in the past, so 👍 on improvements.

Reply:
ha, looks like github was already logging this information, and more, I have updated this pr to match both